### PR TITLE
Adding Fast RTPS documentation

### PR DESCRIPTION
**Pull request by vicentemonge**

### CODE REVIEW

File: en/setup/fast-rtps-installation.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## [Requirements](http://eprosima-fast-rtps.readthedocs.io/en/latest/requirements.html#requirements)
+
+*eProsima Fast RTPS* requires the following packages to work.
+
+### Common Dependencies
+
+#### Gtest
+
+Gtest is needed to compile the tests when building from sources.
```
Comment (LorenzMeier) [inline en/setup/fast-rtps-installation.md]: This is not clear enough. We require in our documentation installation instructions for the major platforms (e.g. apt-get install XYZ or brew install XYZ).

File: en/setup/fast-rtps-installation.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+#### Java
+
+Java is required to make use of our buil-in code generation tool *fastrtpsgen*.
```
Comment (LorenzMeier) [inline en/setup/fast-rtps-installation.md]: Not detailed enough. What version of Java? Is that version already installed with the rest of PX4? (we do have Java as dependency for jMAVSim)

**Review by LorenzMeier — CHANGES_REQUESTED** [review]
Thanks, this is very helpful! To match the quality of the rest of the documentation we need a little more specific instructions in some areas.

File: en/setup/fast-rtps-installation.md (hunk 2 above)
Comment (vicentemonge) [reply en/setup/fast-rtps-installation.md]: We need 1.6 at least but jMAVSim needs 1.8. We will add in the document that we need 1.6 or newest.

Comment (mhkabir) [reply en/setup/fast-rtps-installation.md]: It would make sense just to recommend 1.8. Too many options just confuse people.

Comment (vicentemonge) [reply en/setup/fast-rtps-installation.md]: it seems good.

File: en/setup/fast-rtps-installation.md (hunk 1 above)
Comment (vicentemonge) [reply en/setup/fast-rtps-installation.md]: We think is not necessary since is for test purposes. We'll avoid that in this abstract.

**Review by LorenzMeier — APPROVED** [review]

**Pull request merged**